website/content/docs/auth/azure.mdx

```diff
@@ -234,6 +234,63 @@ tool.
        https://127.0.0.1:8200/v1/auth/azure/role/dev-role
    ```
 
+## Root credential rotation
+
+If the mount is configured with credentials directly, the credential's key may be
+rotated to a Vault-generated value that is not accessible by the operator.
+This will ensure that only Vault is able to access the "root" user that Vault uses to
+manipulate dynamic & static credentials.
+
+```shell-session
+vault write -f auth/azure/rotate-root
+```
+
+### Schedule-based root credential rotation
+
+<EnterpriseAlert product="vault" />
+
+The Azure auth engine supports configuring schedule-based automatic
+credential rotation for the root credential with the
+[rotation_schedule](/vault/api-docs/auth/azure#rotation_schedule) field.
+For example:
+
+```shell-session
+$ vault write auth/azure/config \
+    ...
+    rotation_schedule="0 * * * SAT"
+    ...
+```
+
+This configuration will set the role's credential rotation to occur on Saturday
+at 00:00.
+
+Additionally, this schedule-based approach allows for optionally configuring a
+[rotation_window](/vault/api-docs/auth/azure#rotation_window) in which
+the automatic rotation is allowed to occur. For example:
+
+```shell-session
+$ vault write auth/azure/config \
+    ...
+    rotation_window="1h" \
+    rotation_schedule="0 * * * SAT"
+    ...
+```
+
+This configuration will set rotations to occur on Saturday at 00:00. The 1
+hour `rotation_window` will prevent the rotation from occuring after 01:00. If
+the static role's credential is not rotated during this window, due to a failure
+or otherwise, it will not be rotated until the next scheduled rotation.
```

Comment:
```suggestion
```suggestion
Scheduled root credential rotation can also set a
[rotation_window](/vault/api-docs/auth/azure#rotation_window) during which the
scheduled rotation is allowed to occur. Vault will stop trying to rotate the
credential once the window expires. For example, the following command tells
Vault to rotate the credential on Saturday at midnight, but only within the span
of an hour. If Vault cannot rotate the credential by 1:00, due to a failure
or otherwise, Vault will stop trying to rotate the credential until the next
scheduled rotation.
```

Style correction: write in active voice, avoid possessives, explain code examples before presenting them
```

Reply:
changed in all four plugins